cajita/unit_test/tstSparseIndexSpace.hpp

```diff
@@ -0,0 +1,386 @@
+#include <Cajita_SparseIndexSpace.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace Cajita;
+
+namespace Test
+{
+//---------------------------------------------------------------------------//
+void testAtomicOr()
```

Comment:
@dalg24 This test seems to reproduce the `Kokkos::atomic_fetch_or` failure for HIP on the CI. Could you look at this and let us know if you see anything obvious in this code?